.github/workflows/run-clang-format.sh

```diff
@@ -16,7 +16,7 @@
 FORMAT_OUT=${TMPDIR:-/tmp}/clang-format-diff.out
 
 # Run on all the files that are affected
-clang-format -i --style=file $(git diff --name-only --diff-filter=AM -r origin/master | grep '\(\.cc\|\.h\)$') 2> /dev/null
```

Comment:
It's probably a good idea to make sure clang-format is being run in the root directory.

Reply:
`clang-format` is run from the root directory (the script is invoked as `./.github/workflows/run-clang-format.sh`). But even if it was executed in a different directory, `git diff` would give out relative names (such as `../src/foo.cc`), so everything would still work (and the `.clang-format` file is looked up relative to the source file it is processing not the location the execution. So that als should work).